src/modules/users/CreateUserJob.cpp

```diff
@@ -74,6 +74,8 @@ CreateUserJob::exec()
 
     if ( gs->contains( "sudoersGroup" ) && !gs->value( "sudoersGroup" ).toString().isEmpty() )
     {
+        cDebug() << QString("[CREATEUSER]: preparing sudoers") ;
```

Comment:
That doesn't need to be a QString() at all; generally you'd use QStringLiteral() for this kind of literal (it saves a few cycles and is better with const) but here, you can log a C++ string literal: just drop QString() from there and leave the bare string literal.